modules/programs/direnv.nix

```diff
@@ -162,28 +162,20 @@ in {
           $env.config.hooks.pre_prompt?
           | default []
           | append {||
-              let direnv = (
-                  ${direnvWrapped}
-                  | from json --strict
-                  | default {}
-              )
-              if ($direnv | is-empty) {
-                  return
-              }
-              $direnv
+              ${direnvWrapped}
+              | from json --strict
+              | default {}
               | items {|key, value|
-                  {
-                      key: $key
-                      value: (do (
-                          $env.ENV_CONVERSIONS?
-                          | default {}
-                          | get -i $key
-                          | get -i from_string
-                          | default {|x| $x}
-                      ) $value)
-                  }
+                let value = do (
+                  $env.ENV_CONVERSIONS?
+                  | default {}
+                  | get -i $key
+                  | get -i from_string
+                  | default {|x| $x}
+                ) $value
+                return [ $key $value ]
```

Comment:
I was about to ask the reason for the explicit `return`, but I guess explicitness doesn't hurt :)

Reply:
I would always prefer explicit returns. But, I'm not familiar with nu standards.

Reply:
I don't have a strong preference, it just had a bit too many sigils in a row for my taste without it.